lib/ansible/config/base.yml

```diff
@@ -401,6 +401,18 @@ LOCALHOST_WARNING:
   - {key: localhost_warning, section: defaults}
   type: boolean
   version_added: "2.6"
+IMPLICIT_LOCALHOST_WARNING:
```

Comment:
Having both a LOCALHOST_WARNING and an IMPLICIT_LOCALHOST_WARNING seems confusing (especially since they're both to silence implicit localhost warnings). Maybe a INVENTORY_UNPARSED_WARNING setting would be clearer for what this is actually doing.

Reply:
Agreed. Updated to use this text instead.